packages/core/src/Stores/notification-store.js

```diff
@@ -406,6 +409,10 @@ export default class NotificationStore extends BaseStore {
                 this.addNotificationMessage(this.client_notifications.has_changed_two_fa);
             }
 
+            if (this.root_store.client.should_show_passkey_notification) {
+                this.addNotificationMessage(this.client_notifications.enable_passkey);
+            }
```

Comment:
Are we showing notification to desktop users as well?
shouldnt we check isMobile before adding passkeys notification?

Reply:
yes, we are checking it on line 463 before running the logic to set the notification.